third_party/houdini/lib/gusd/stageCache.cpp

```diff
@@ -56,21 +56,39 @@
 
 PXR_NAMESPACE_OPEN_SCOPE
 
-
-TF_DEFINE_ENV_SETTING(GUSD_STAGEMASK_EXPANDRELS, true,
-                      "Expand stage masks to include targets of relationships. "
-                      "It may be possible to disable this option, which may "
-                      "provide performance gains, but correctness cannot be "
-                      "guaranteed when doing so.");
-
-
-TF_DEFINE_ENV_SETTING(GUSD_STAGEMASK_ENABLE, true,
-                      "Enable use of stage masks when accessing prims from "
-                      "the cache. Note that disabling this feature may "
-                      "be very detrimental to performance when separately "
-                      "querying many prims with variant selections "
-                      "(or other types of stage edits).");
-
+// The follofing declarations are equivalent to using the TF_DEFINE_ENV_SETTING
```

Comment:
Hey @marktucker, I'm looking at merging this pull request in now and was looking at these changes. Could you give any more details about how you ran into this issue -- what platform, etc? I think we'd like to pull this fix into tf/envSetting.h itself rather than have it here, but I was hoping to get a little more context. Thanks!

Reply:
Thanks Mark, that did help quite a bit. I'm testing a change that folds your fix into tf now. One other question: what compiler were you using when you ran into this? Were you using Visual Studio 2017?

Reply:
Visual Studio 2017. Version 15.7.4 to be precise.

Reply:
OK, that seems to line up with what I've been seeing in godbolt. From the following thread, it sounds like this could be a bug in MSVC 2017:

https://developercommunity.visualstudio.com/content/problem/76198/vs-2017-compiler-creates-broken-debug-build-using.html

In the meantime, I've adopted your fix into tf/envSetting.h internally and will push that bit out in a separate change. Thanks again!